src/Symfony/Component/Inflector/Tests/InflectorTest.php

```diff
@@ -58,7 +58,7 @@ public function singularizeProvider()
             ['crises', ['cris', 'crise', 'crisis']],
             ['criteria', ['criterion', 'criterium']],
             ['cups', 'cup'],
-            ['data', ['daton', 'datum']],
```

Comment:
Why was this test changed?

Previously issues with singular `datum` were considered as a "feature" - https://github.com/symfony/symfony/issues/17764#issuecomment-183864898